src/app/components/mno-offers-list/mno-offers-list.coffee

```diff
@@ -0,0 +1,34 @@
+#
+# Nested Table Expand Directive
+#
+
+@App.directive('mnoOffersList', ($translate, $filter) ->
+  restrict: 'AE'
+  scope: {
+    offers: '='
+    markup: '='
+  },
+  templateUrl: 'app/components/mno-offers-list/mno-offers-list.html',
+  link: (scope, elem, attrs) ->
+    scope.formatPrice = (priceHash) ->
```

Comment:
Should all this price formatting logic moved to the backend?

If not, we're probably doing it elsewhere in the admin panel and it'd be good to extract everything into a money/price helper if it doesn't exist already.